ceph/tests/test_unit.py

```diff
@@ -90,7 +94,17 @@ def test_tagged_metrics(_, aggregator):
     for pool in ['pool0', 'rbd']:
         expected_tags = EXPECTED_TAGS + ['ceph_pool:%s' % pool]
 
-        for metric in ['ceph.read_bytes', 'ceph.write_bytes', 'ceph.pct_used', 'ceph.num_objects']:
+        for metric in [
+            'ceph.read_bytes',
+            'ceph.write_bytes',
+            'ceph.pct_used',
+            'ceph.num_objects',
+            'ceph.misplaced_objects',
+            'ceph.misplaced_objects',
+            'ceph.recovering_objects_per_sec',
+            'ceph.recovering_bytes_per_sec',
+            'ceph.recovering_keys_per_sec',
+        ]:
```

Comment:
Small nit but since this list is getting long, it would be better to move it to the top and call it `EXPECTED_METRICS_TAG_POOL` or something similar.